src/main.h

```diff
@@ -398,7 +398,7 @@ void UpdateCoins(const CTransaction& tx, CValidationState& state, CCoinsViewCach
 
 /** Context-independent validity checks */
 bool CheckTransaction(const CTransaction& tx, bool fZerocoinActive, bool fRejectBadUTXO, CValidationState& state, bool fWitnessActive);
```

Comment:
Fixed the compile error on Mac. I left the zerocoin parameter in `CheckTransaction` but renamed it. It helps to keep align with the upstream.
Which text should be in different color on the overview? There is GUI overhaul in the upstream, and I changed to keep the color in Send as the original design in Send.
We didn't recalculate supply for zerocoin.

Reply:
> Which text should be in different color on the overview? There is GUI overhaul in the upstream, and I changed to keep the color in Send as the original design in Send.

The transactions text on the top right in the above image is practically unreadable due to it being white. This is also happening on the transactions tab as well.

![Screen Shot 2021-05-23 at 10 28 03 PM](https://user-images.githubusercontent.com/13294167/119300765-2a4e2980-bc16-11eb-9a2e-8938da92e606.png)

Reply:
I pushed a commit trying to fix the text color.
However, I can't see any changes of the text color between the PR and the original code. So I'm not sure what's wrong. Also, since I don't have any transactions, I can't reproduce this problem on my side.